editoast/src/core/v2/pathfinding.rs

```diff
@@ -35,6 +40,36 @@ pub struct PathfindingRequest {
     pub rolling_stock_supported_signaling_systems: Vec<String>,
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize, ToSchema)]
+pub struct RangeOffset {
+    start: u64,
+    end: u64,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize, ToSchema)]
+pub struct IncompatibleElectrification {
+    range: RangeOffset,
+    value: String,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize, ToSchema)]
+pub struct IncompatibleLoadingGauge {
+    range: RangeOffset,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize, ToSchema)]
+pub struct IncompatibleSignalingSystem {
+    range: RangeOffset,
+    value: String,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize, ToSchema)]
+pub struct IncompatibleConstraints {
+    incompatible_electrification_ranges: Option<Vec<IncompatibleElectrification>>,
+    incompatible_gauge_ranges: Option<Vec<IncompatibleLoadingGauge>>,
+    incompatible_signalisation_system_ranges: Option<Vec<IncompatibleSignalingSystem>>,
```

Comment:
This doesn't match core and shadows the actual value (leading to always `null` list).
To be done in multiple places:
```suggestion
    incompatible_signaling_system_ranges: Option<Vec<IncompatibleSignalingSystem>>,
```